driver-core/src/main/com/mongodb/ConnectionString.java

```diff
@@ -1127,14 +1127,16 @@ public WriteConcern getWriteConcern() {
     /**
      * Returns true if writes should be retried if they fail due to a network error, and false otherwise
      *
-     * @return the retryWrites value, or false if unset
+     * <p>Starting with the 3.11.0 release, the default value is true</p>
+     *
+     * @return the retryWrites value, or true if unset
```

Comment:
Note the class level javadoc needs the default changing.

Reply:
Done